x/protocolpool/keeper/genesis.go

```diff
@@ -96,10 +96,15 @@ func (k Keeper) ExportGenesis(ctx context.Context) (*types.GenesisState, error)
 
 	genState := types.NewGenesisState(cf, budget)
 
-	genState.ToDistribute, err = k.ToDistribute.Get(ctx)
+	balances := k.bankKeeper.GetAllBalances(ctx, k.authKeeper.GetModuleAccount(ctx, types.ProtocolPoolDistrAccount).GetAddress())
+	bondDenom, err := k.stakingKeeper.BondDenom(ctx)
 	if err != nil {
 		return nil, err
 	}
 
+	genState.ToDistribute = balances.AmountOf(bondDenom)
+
+	// TODO: fix export by adding all the funds to be distributed
```

Comment:
will this be completed in a follow up pr?

Reply:
Just added it here: https://github.com/cosmos/cosmos-sdk/pull/20790/commits/4dc77b153bdb2367fb161715a44f0c62ee7c0e92#diff-443d2bd72ab97253021289109c54b93f9e87f182481df4a6cdfa5a18fff20ae0